.travis.yml

```diff
@@ -33,13 +33,7 @@ jobs:
       after_success:
         - rm -rf $TRAVIS_BUILD_DIR/.tmp
         - bash <(curl -s https://codecov.io/bash)
-    - addons:
-        sauce_connect: true
-      script:
-        - npm run cluster
-        - npm run test:e2e
-      env:
-        - KUBE_DASHBOARD_KUBECONFIG=$HOME/.kube/config
+    - script: npm run test:e2e
```

Comment:
Why dropping `sauce_connect`?

Reply:
We aren't using sauce_labs for our tests anymore.

If we plan to use them again, I'll revert, but they were also the cause of a lot of flakes so I'd opt against. 

Let me know your thoughts!

Reply:
This is quite weird because in the protractor conf we do have config for sauce labs.

https://github.com/kubernetes/dashboard/blob/1bbe49858d461d8f11eb4fbed126c0cf6480b5f0/aio/protractor.conf.js#L38-L63

Those variables are also defined in the travis settings.

Reply:
I had checked several job logs for any sign of Sauce Labs running and found none. It looks like no actual tests are running. 

Example: https://api.travis-ci.org/v3/job/486470328/log.txt (this is off of #3542) 

```
[15:24:50] I/update - chromedriver: chromedriver_2.45 up to date
[15:24:51] I/launcher - Running 1 instances of WebDriver
[15:24:51] I/local - Starting selenium standalone server...
[15:24:53] I/local - Selenium standalone server started at http://10.20.4.242:47825/wd/hub
Jasmine started

Executed 0 of 0 specs[32m SUCCESS[39m in 0.005 sec.
[15:24:58] I/local - Shutting down selenium standalone server.
[15:24:58] I/launcher - 0 instance(s) of WebDriver still running
[15:24:58] I/launcher - chrome #01 passed
```

If we're still intending to use Sauce Labs I'll revert that change and keep moving forward. :)